html/elements/meta.json

```
@@ -675,10 +675,12 @@
                   "version_added": false
                 },
                 "safari": {
-                  "version_added": false
+                  "version_added": "15",
+                  "notes": "Safari 15 beta supports a media attribute that recognizes (prefers-color-scheme: light) or (prefers-color-scheme: dark)."
```

A few things on these notes, in order of seriousness:

1. I don't understand this note. The way this is phrased makes it seem quite indirect. I don't understand how it relates to the feature. A media attribute? Which one?
2. Since _only_ version 15 of Safari is supported, there's no need to mention the status (which is also captured in the release data) nor the version number. This helps prevent notes from going stale.
3. You can use a subset of HTML in notes, including `<code>`.

```suggestion
                  "notes": "Safari supports a media attribute that recognizes <code>prefers-color-scheme: light</code> or <code>prefers-color-scheme: dark</code>."
```

---

Perhaps the better alternative proposal is to add a new entry for `media` showing Safari 15 support with a status of `experimental` and `standard_track`. This note could be attached there and rewritten to describe support for these specific cases.

---

It's probably best to move `media` attribute support to a new entry.

---

We're going to hold off updating `media` attribute in a follow-up PR.